lisa/util/process.py

```diff
@@ -35,11 +35,13 @@
     re.compile(r"Password: .+\r\nsudo: timed out reading password"),
 ]
 
+ANSI_ESCAPE = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
```

Comment:
There is an util method in utils already. Please filter out it in the place, instead of the lower level class. I'm not sure if there are side effects on existing code.

Reply:
I made the changes as you requested. However, my fear is that we will waste a lot of time debugging issues related to regex matching being affected by ANSI escape characters. These issues are not obvious from the error messages and take time to debug. We shouldn't rely on LISA contributors to foresee ANSI characters in stdout. It should be easy to write bug free code that works on all distros.

Reply:
Sorry, I forgot it. I agree to add it in execute result. @LiliDeng, please be aware check if there are regressions.

Reply:
Okay, I will move it back to the executable result.